core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -144,7 +144,7 @@ private RpcUtils() {} // prevent instantiation
         }
       }
       throw AlluxioStatusException.fromIOException(e).toGrpcStatusException();
-    } catch (RuntimeException e) {
+    } catch (RuntimeException | LinkageError e) {
```

Comment:
I purposely made it restrictive instead of catching throwable because it causes backwards incompatiblilty otherwise.

Reply:
document here why we have linkagerror to catch?
also do you want to provide a logging with a bit more hint on what's going on here?

Reply:
the exception gets logged, that is the important part. Before this change, Linkageerrors such as classnotfound are simply silently failing, making it hard to debug. 
Linkage errors tend to come from UFS libs.